data/src/test/java/io/gnosis/data/repositories/TransactionRepositoryTest.kt

```diff
@@ -155,7 +156,14 @@ class TransactionRepositoryTest {
     fun `getTransactions (multisig transaction for ERC721) should return transfer`() = runBlockingTest {
         val transactionDto = buildMultisigTransactionDto(
             contractInfoType = ContractInfoType.ERC721,
-            dataDecodedDto = DataDecodedDto("safeTransferFrom", null) // TODO: check also transferFrom (might have differrent adress copied
+            dataDecodedDto = DataDecodedDto(
+                "safeTransferFrom",
+                listOf(
+                    ParamsDto("from", "address", defaultFromAddress.asEthereumAddressChecksumString()),
+                    ParamsDto("to", "address", defaultToAddress.asEthereumAddressChecksumString()),
+                    ParamsDto("tokenId", "uint256", "23")
```

Comment:
why is `tokenId` set to 23 here? erc721 transfer json has a different value

Reply:
"23" is just a random value for me. Like "foo" or "bar". Maybe "<random value>" is better? Or a `val defaultTokenId = "23"`?

Reply:
Extracted to defaultTokenId in b4c2b8a8e90f167fd41626ac300cc715b5b38476